app/src/main/java/com/g11x/checklistapp/ImportantInformationActivity.java

```diff
@@ -29,12 +33,18 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
+import com.g11x.checklistapp.data.Database;
+
 import java.util.ArrayList;
 
 /**
  * Display a list of important information items and provide the ability to create a new one.
  */
 public class ImportantInformationActivity extends NavigationActivity {
+  private static final String[] PROJECTION = {
```

Comment:
+1 This is a good pattern to follow, we should encourage others to do so as well.